DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/CentreManagerDetails.cshtml

```diff
@@ -0,0 +1,39 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model CentreManagerDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = errorHasOccurred ? "Error: Centre Manager Details" : "Centre Manager Details";

```

Comment:
Do we want this capitalised like this?

Reply:
Looks like a vast majority of the ViewData titles have the first letter of each word capitalised. There are only a few titles where we only have the first first letter capitalised, and those tend to be in the recent additions of my own where I got confused between the capitalisation styles of the Title and the body. So I think we should prioritise consistency and keep the first letter of each word in titles capitalised. I'll probably have to go back in future to change the others at a later date.